pkgs/build-support/setup-hooks/copy-desktop-items.sh

```diff
@@ -30,12 +30,14 @@ copyDesktopItems() {
 
     for desktopItem in $desktopItems; do
         if [[ -f "$desktopItem" ]]; then
-            echo "Copying '$desktopItem' into '$out/share/applications'"
-            install -D -m 444 -t "$out"/share/applications "$desktopItem"
+            echo \
+              "Copying '$desktopItem' into '${!outputBin}/share/applications'"
```

Comment:
Maybe stick this in a variable to avoid repeating it everywhere?

Reply:
Definitely cleaner 👍